### PR TITLE
Updated variable name from myName to ourName

### LINKED ISSUE #5354: JavaScript - Waypoint: Declare JavaScript Variables bug
The code windows has used 'ourName' as the variable. But the explanation on the left window says 'myName' as the variable.

### PR DESCRIPTION
**Pull request by bogadodiegoh**
Changed description text and test section to use ourName instead of
myName, in basic-javascript.json file.

Closes #5354.

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 1]
```diff
@@ -104,7 +104,7 @@
       "id": "bd7123c9c444eddfaeb5bdef",
       "title": "Declare String Variables",
       "description": [
-        "In the previous challenge, we used the code <code>var myName = \"your name\"</code>. This is what we call a <code>String</code> variable. It is nothing more than a \"string\" of characters. JavaScript strings are always wrapped in quotes.",
+        "In the previous challenge, we used the code <code>var ourName = \"your name\"</code>. This is what we call a <code>String</code> variable. It is nothing more than a \"string\" of characters. JavaScript strings are always wrapped in quotes.",
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: This and the translation of it does not need to be changed as the previous challenge will have exposed the user to a `myName` variable once they have completed it.

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 2]
```diff
@@ -73,13 +73,13 @@
...
-        "Now, use the <code>var</code> keyword to create a variable called <code>myName</code>. Set its value to your name, in double quotes.",
+        "Let's create our first variable and call it \"ourName\".",
+        "You'll notice that in <code>ourName</code>, we didn't use a space, and that the \"N\" is capitalized. JavaScript variables are written in <code>camel case</code>. An example of camel case is: camelCase.",
+        "Now, use the <code>var</code> keyword to create a variable called <code>ourName</code>. Set its value to your name, in double quotes.",
         "Look at the <code>ourName</code> example if you get stuck."
       ],
       "tests": [
-        "assert((function(){if(typeof myName !== \"undefined\" && typeof myName === \"string\" && myName.length > 0){return true;}else{return false;}})(), 'message: <code>myName</code> should be a string that contains at least one character in it.');"
+        "assert((function(){if(typeof ourName !== \"undefined\" && typeof ourName === \"string\" && ourName.length > 0){return true;}else{return false;}})(), 'message: <code>ourName</code> should be a string that contains at least one character in it.');"
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: The tests should still be looking at the `myName` variable.

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 3]
```diff
@@ -87,16 +87,16 @@
         ""
       ],
       "tail": [
-        "if(typeof myName !== \"undefined\"){(function(v){return v;})(myName);}"
+        "if(typeof ourName !== \"undefined\"){(function(v){return v;})(ourName);}"
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: Display function should still be looking for `myName`

**Comment by ltegman** [comment]
Thanks for the PR! Looks like you changed a few too many references to the `myName` variable. Many of them should still be left intact. After you change back the requested references you can `git add . && git commit --amend && git push -f` to update this PR.

File: seed/challenges/01-front-end-development-certification/basic-javascript.json [hunk 4]
```diff
@@ -73,9 +73,9 @@
       "title": "Declare JavaScript Variables",
       "description": [
         "When we store data in a <code>data structure</code>, we call it a <code>variable</code>. These variables are no different from the x and y variables you use in math.",
-        "Let's create our first variable and call it \"myName\".",
-        "You'll notice that in <code>myName</code>, we didn't use a space, and that the \"N\" is capitalized. JavaScript variables are written in <code>camel case</code>. An example of camel case is: camelCase.",
-        "Now, use the <code>var</code> keyword to create a variable called <code>myName</code>. Set its value to your name, in double quotes.",
+        "Let's create our first variable and call it \"ourName\".",
+        "You'll notice that in <code>ourName</code>, we didn't use a space, and that the \"N\" is capitalized. JavaScript variables are written in <code>camel case</code>. An example of camel case is: camelCase.",
+        "Now, use the <code>var</code> keyword to create a variable called <code>ourName</code>. Set its value to your name, in double quotes.",
```
Comment (ltegman) [inline seed/challenges/01-front-end-development-certification/basic-javascript.json]: This and the translated version should also still be `myName`. Thanks for spending the time to correct this.

**Comment by BerkeleyTrue** [comment]
closing as stale. Please feel free to resubmit when ready.

Happy Coding!

**Pull request closed without merging**